backend/core/src/listings/entities/listing.entity.ts

```diff
@@ -668,6 +668,20 @@ class Listing extends BaseEntity {
   @ValidateNested({ groups: [ValidationsGroupsEnum.default], each: true })
   @Type(() => ListingUtilities)
   utilities?: ListingUtilities
+
+  @Column({ type: "timestamptz", nullable: true, default: "1970-01-01" })
```

Comment:
I'm not seeing other Date columns with a default, just curious why this column has this default

Reply:
It has a default set to the beginning of time so that it'll be processed by the check the first time it runs.